server/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "ark-automate",
   "version": "1.0.0",
   "scripts": {
-    "start": "node server",
+    "start": "node server.js",
```

Comment:
I am currently not able to test if this works. Why is this change needed?

Reply:
You are right. I changed it to just server.